pdg/src/graph.rs

```diff
@@ -15,47 +15,85 @@ use crate::util::ShortOption;
 
 #[derive(Debug, Eq, PartialEq, Hash, Clone)]
 pub enum NodeKind {
-    /// A copy from one local to another.  This also covers casts such as `&mut T` to `&T` or `&T`
-    /// to `*const T` that don't change the type or value of the pointer.
+    /// A [`Copy`](Self::Copy) from one [`Local`] to another.
```

Comment:
i think it's fine to refer to "copy" colloquially here, no need to self-reference

Reply:
Okay that makes sense.  I'll change that back.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/658/commits/40f6ecfd73752c804da9be32c67d79acc36dcade.